tests/integration/config.go

```diff
@@ -300,7 +300,8 @@ func MultiConsumerTestRun() TestRun {
 					".app_state.slashing.params.signed_blocks_window = \"2\" | " +
 					".app_state.slashing.params.min_signed_per_window = \"0.500000000000000000\" | " +
 					".app_state.slashing.params.downtime_jail_duration = \"2s\" | " +
-					".app_state.slashing.params.slash_fraction_downtime = \"0.010000000000000000\"",
+					".app_state.slashing.params.slash_fraction_downtime = \"0.010000000000000000\" | " +
+					".app_state.provider.params.slash_meter_replenish_fraction = \"1.0\"",
```

Comment:
Hey, can you include this in all test runs?

`MultiConsumerTestRun`
`DemocracyTestRun`
`DefaultTestRun`

Reply:
Solid suggestion, done